chain-tx-enclave-next/mls/src/ciphersuite.rs

```diff
@@ -296,15 +292,88 @@ impl CipherSuite {
     }
 
     /// TODO: use generic array?
-    pub fn hash(&self, data: &[u8]) -> Vec<u8> {
+    pub fn hash(self, data: &[u8]) -> Vec<u8> {
```

Comment:
this is fine, not necessary to use generic array

Reply:
generic array would prevent some problems at compile-time, because there are various `Vec<u8>` (hashes, secrets, nonces, pubkeys...) which can be quite error-prone